rate_counter/rate/models.py

```diff
@@ -1,5 +1,40 @@
 from __future__ import unicode_literals
+from django.contrib.auth.models import User
 
 from django.db import models
 
-# Create your models here.
+
+class Subject(models.Model):
+    """
+    Subject model representing subject from University schedule
+    """
+    name = models.CharField(max_length=100)
+    group = models.ForeignKey('StudentGroup', related_name='subjects')
+
+
+class Institute(models.Model):
+    """
+    Institute model representing Institute parsed from University
+    """
+    name = models.CharField(max_length=4)
+
```

Comment:
Do you mean for second parsing? because first time you will just got it from site i guess